dace/config_schema.yml

```diff
@@ -485,6 +485,13 @@ required:
                         default: false
                         title: Verilator Enable Debug
                         description: Enable/disable verbose internal state debug output.
+                    mode:
+                        type: str
+                        default: simulator
+                        title: Compilation mode
+                        description: >
+                          Target of FPGA kernel build
+                          Can be either (simulator/xilinx).
```

Comment:
Why not use the existing mode of Xilinx?

Reply:
Sure, I guess I could check whether it's in device in Python and whether the file targets Xilinx in CMake?

Reply:
I thought we passed this to CMake already actually, if not just pass it as well :-)

Reply:
Still seems to me like this is redundant with `fpga_vendor` + `xilinx_mode`

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/4a7e567b636a6801f476fc97d8e6b7d74acde946